docs/src/modules/components/AppSettingsDrawer.js

```diff
@@ -42,44 +43,25 @@ function AppSettingsDrawer(props) {
   const t = useTranslate();
   const upperTheme = useTheme();
   const changeTheme = useChangeTheme();
-  const [mode, setMode] = React.useState(null);
+  const [mode, setMode] = useLocalStorageState('mui-mode', 'system');
```

Comment:
Ideally, this state is lifted up to where `changeTheme` is created as to avoid updating it in an effect